.github/workflows/ensurepip_verify_bundled.yml

```diff
@@ -0,0 +1,51 @@
+name: Verify bundled pip and setuptools
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'Lib/ensurepip/_bundled/**'
+  pull_request:
+    paths:
+      - 'Lib/ensurepip/_bundled/**'
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Compare checksums of bundled pip and setuptools to ones published on PyPI
+        run: |
+          package_names=("pip" "setuptools")
```

Comment:
Perhaps this could be moved to a script (say, under `Tools/scripts`), so it can also be run locally?

Reply:
Good idea, but it looks like all scripts in `Tools/scripts` are written in Python.
This is a Bash script, and it may not work on some platforms.

Do you know how compatible with different platforms scripts located in `Tools/scripts` should be?

Reply:
Hmm, or perhaps the `Misc` directory? The [devguide](https://devguide.python.org/?highlight=misc#additional-resources) says:

> Various tools with configuration files as found in the [Misc directory](https://github.com/python/cpython/tree/main/Misc)

I don't know what compatibility is needed, but as this script is primarily intended for Ubuntu on the CI, I'd stick with that.

Reply:
Done. [This is](https://github.com/illia-v/cpython/runs/5557015587?check_suite_focus=true) the latest run after I corrupted the pip wheel temporarily.

BTW, I added messages that GitHub has to show as annotations. But I could not see the annotations, maybe it does not show them for binary files…